src/Microsoft.AspNet.Mvc.ModelBinding/Metadata/IModelMetadataBindingDetailsProvider.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.Framework.Internal;
+
+namespace Microsoft.AspNet.Mvc.ModelBinding
+{
+    public interface IModelMetadataBindingDetailsProvider : IModelMetadataDetailsProvider
+    {
+        void GetBindingDetails([NotNull] ModelMetadataBindingDetailsContext context);
```

Comment:
I am not sure if its only me or does it look weird a Get* method returning void. Should we drop the ModelMetadataBindingDetails property from the context and have that as a return parameter here?